scripts/build-wasm.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2021 Adobe
+# All Rights Reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -e
+
+rm -rf build/wasm
+mkdir -p build/wasm
+
+CMAKE_ARGS=()
+
+# CMake-level configuration
+CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Release")
+CMAKE_ARGS+=("-DXNNPACK_LIBRARY_TYPE=static")
+
+if [ "$1" == "enable_tests" ] || [ "$2" == "enable_tests" ] || [ "$3" == "enable_tests" ] || [ "$4" == "enable_tests" ]; then
```

Comment:
Instead of defining your own command-line arguments to enable tests and benchmarks, pass command-line arguments directly to `cmake`, i.e. https://github.com/google/XNNPACK/blob/master/scripts/build-local.sh#L30-L31
Then the script can be invoked with e.g. `scripts/build-wasm.sh -DXNNPACK_BUILD_TESTS=OFF`, similarly to other `scripts/build-*.sh` scripts